docs/migration.md

```diff
@@ -102,6 +105,10 @@ consider additional positioning prop support on a case-by-case basis.
   - same breaking changes as `ColorSwatch`.
   - `popperModifiers` prop (see [note](#breaking-changes))
 
+<!-- prettier-ignore -->
```

Comment:
Prettier [doesn't support](https://github.com/prettier/prettier/issues/15479) Github-flavored Markdown [alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts). 😢